index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon - SEO Experts</title>
+  </head>
+
+  <body>
+    <!--section containing header and nav bar embedded within containing links-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- section containing nav with individual links embedded leading to different parts of the site-->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimizatoin">Search Engine Optimization</a>
```

Comment:
This is the link that is broken which I mentioned. The other 2 work fine, see if you can find out why this one doesnt and correct it.